package.json

```diff
@@ -1,139 +1,144 @@
-            {
-                "language": "pawn",
-                "path": "./snippets/macros.json"
-            },
-            {
-                "language": "pawn",
-                "path": "./snippets/functions-standard.json"
-            },
-            {
-                "language": "pawn",
-                "path": "./snippets/functions-community.json"
-            },
-            {
-                "language": "pawn",
-                "path": "./snippets/definitions-standard.json"
-            },
-            {
-                "language": "pawn",
-                "path": "./snippets/definitions-community.json"
-            }
-        ],
-        "problemMatchers": [
-            {
-                "name": "pawncc",
-                "owner": "pawn",
-                "fileLocation": [
-                    "relative",
-                    "${workspaceRoot}/gamemodes"
-                ],
-                "pattern": {
-                    "regexp": "^(.*?)\\(([0-9]*)[- 0-9]*\\) \\: (fatal error|error|warning) [0-9]*\\: (.*)$",
-                    "file": 1,
-                    "location": 2,
-                    "severity": 3,
-                    "message": 4
-                }
-            },
-            {
-                "name": "sampctl",
-                "owner": "pawn",
-                "fileLocation": "absolute",
-                "pattern": [
-                    {
-                        "regexp": "^(.+):([0-9]+) \\((\\w+)\\) (.+)$",
-                        "file": 1,
-                        "location": 2,
-                        "severity": 3,
-                        "message": 4
-                    }
-                ],
-                "background": {
-                    "activeOnStart": true,
-                    "beginsPattern": "watch-build: starting compilation",
-                    "endsPattern": "watch-build: finished"
-                }
-            }
-        ],
-        "commands": [
-            {
-                "command": "extension.packageEnsure",
-                "title": "Ensure Package Dependencies"
-            },
-            {
-                "command": "extension.packageBuild",
-                "title": "Build Pawn Package"
-            }
-        ]
-    },
-    "scripts": {
-        "vscode:prepublish": "npm run compile",
-        "update-vscode": "node ./node_modules/vscode/bin/install",
-        "compile": "tsc -p ./",
-        "watch": "tsc -watch -p ./",
-        "postinstall": "node ./node_modules/vscode/bin/install",
-        "test": "npm run compile && node ./node_modules/vscode/bin/test"
-    },
-    "devDependencies": {
-        "typescript": "^4.1.2",
-        "vscode": "^1.1.36",
-        "@types/node": "^14.14.11",
-        "@types/mocha": "^8.2.0"
-    },
-    "dependencies": {
-        "axios": "^0.21.0",
-        "vscode-languageclient": "^6.1.3",
-        "vscode-languageserver": "^6.1.1",
-        "vscode-languageserver-textdocument": "^1.0.1"
-    }
+	"name": "vscode-pawn",
```

Comment:
accidental change? should be reverted, or cleaned up if there is an actual change you made to this file.